canvas_modules/common-canvas/src/notification-panel/notification-panel.scss

```diff
@@ -82,11 +86,11 @@ $toolbar-button-height: 41px; // Allow one extra pixel for the toolbar border
 
 .notification-panel-messages-container {
 	width: inherit;
-	max-height: calc(100% - #{$notification-panel-header-height});
-	height: fit-content;
+	max-height: calc(100% - #{$notification-panel-header-height} - 10px); // Extra 10px makes sure full height of notification-panel-button-container is visible in narrow window
```

Comment:
Is the 10px here just an arbitrary number? If it is the button height can we create a variable set to 10px with an appropriate name so the calculation expression is readable?  

Also, although the word 'narrow' is used in the issue title for this work it is the wrong word since narrow refers to a a small , or constrained, width of something. I don't think there is an English equivalent of narrow in the vertical direction but maybe 'short' would be a better word.

Thirdly, did you experiment with `vh` values for this instead of a calculation using pixels -- as mentioned in my comment in the issue?

Reply:
> Is the 10px here just an arbitrary number? If it is the button height can we create a variable set to 10px with an appropriate name so the calculation expression is readable?

> Also, although the word 'narrow' is used in the issue title for this work it is the wrong word since narrow refers to a a small , or constrained, width of something. I don't think there is an English equivalent of narrow in the vertical direction but maybe 'short' would be a better word.

Removed 10px and used `$notification-panel-subtitle-height`. Removed comment because `calc` expression is readable.


> Thirdly, did you experiment with vh values for this instead of a calculation using pixels -- as mentioned in my comment in the issue?

Yes. I wasn't able to fix this issue using `vh` values. 
I see height 100% is also used in parent div `common-canvas-items-container` class. Followed the similar height 100% calculation for notification-panel.
![Screenshot 2023-10-16 at 2 04 49 PM](https://github.com/elyra-ai/canvas/assets/25124000/3775b99a-667a-4e37-8704-8b4367f209c5)